pyannote/core/timeline.py

```diff
@@ -1130,4 +1131,7 @@ def _repr_png_(self):
         """
 
         from .notebook import repr_timeline
-        return repr_timeline(self)
+        try:
+            return repr_timeline(self)
+        except ImportError:
+            return None
```

Comment:
Same. Would be nice to warn users about the existence of pyannote.core[notebook].